packages/plugin-layout/src/LayoutEngine.ts

```diff
@@ -8,7 +8,7 @@ export type DomZonePosition = 'after' | 'prepend' | 'append' | 'replace';
 
 export interface ComponentDefinition {
     id: ComponentId;
-    render: (editor: JWEditor) => Promise<VNode[]>;
+    render: (editor: JWEditor, params?: Record<string, any>) => Promise<VNode[]>;
```

Comment:
Your linter let you do that ? That's weird because we're supposed to have a shared configuration and ours does not let us write `any`. We usually write `{}` instead. Like `any` it matches everything, but it will generate an error if you try to use it in any meaningful way.

With any, you can do that:
```ts
var a: Any = { a: 'a' };
console.log(a.b); // no warning
```

With `{}`, you can't do that:
```ts
var a: {} = { a: 'a' };
console.log(a.b); // warning: `{}` does not have a `b` property
```

Reply:
Nice to know, I will change it